### ISSUE TITLE
There is a typo in the Doctrine driver

### ISSUE THREAD

**Issue by TFarla** [bernardphp/bernard#265]
## Problem
The doctrine driver fails because there is a space missing in this query:
```
$query = 'SELECT id, message FROM bernard_messages
                  WHERE queue = :queue AND visible = :visible
                  ORDER BY sentAt, id LIMIT 1' . $this->connection->getDatabasePlatform()->getForUpdateSql();
```

Location: DoctrineDriver.php:104

**Comment by danydev** [comment]
fixed on master, see #240, #223

**Comment by acrobat** [comment]
Fixed on master (next alpha will be tagged soon)
